### DISCUSSION TITLE
UHGGv2 + UHGV MQ+ database with Bracken database for 100bp reads

### DISCUSSION THREAD

**Discussion by CarsonJM** [bhattlab/phanta#41] [Phanta DBs] [answered]
Hi there, thank you for the great tool! I am wondering if it would be possible to supply a 100bp Bracken database (or the necessary input files) for the UHGGv2 + UHGV MQ+ version. 

I have used the original database, which has the capability to take 100bp reads, but think the cluster based family assignments in UHGV will be much more informative than the old morphology-based (i.e. Siphoviridae, Myoviridae... etc) taxonomies in MGV. 

Thank you!

**Comment by meenachakra** [comment]
Hi Carson, definitely! Will supply by Friday.

**Reply by CarsonJM** [reply]
Fantastic, thank you for the quick reply!

**Reply by meenachakra** [reply, marked as answer]
No worries, here you go!
[database for carson.zip](https://github.com/bhattlab/phanta/files/13984567/database.for.carson.zip)

**Comment by rozwalak** [comment]
Hi, 

thank you for prepare version for 100bp. Could you add this file (and maybe for other read lengths as well) to the folder here https://github.com/bhattlab/phanta/blob/main/databases.md.  

I would like to include a link to downloading the database in my pipeline, and including everything there will help me a lot with reproducibility. 

Thanks in advance! 

Best, 
Piotr

**Reply by meenachakra** [reply]
Hi Piotr,
Sure, will set a reminder to do next week! :)

**Reply by yipinto** [reply]
Hi Piotr, 
Done, check the updated link.
Yishay
